docs/reference/query-languages/esql/functions-operators/mv-functions.md

```diff
@@ -68,6 +68,9 @@ mapped_pages:
 :::{include} ../_snippets/functions/layout/mv_sum.md
 :::
 
+:::{include} ../_snippets/functions/layout/mv_union.md
```

Comment:
We also need to add MV_UNION to this list: [_snippets/lists/mv-functions.md](https://github.com/elastic/elasticsearch/blob/main/docs/reference/query-languages/esql/_snippets/lists/mv-functions.md) with an inline applies_to tag :)

That way the new function appears in the list at the start of the page